docs/UserGuide.md

```diff
@@ -45,13 +49,13 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 **:information_source: Notes about the command format:**<br>
 
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/Meta`.
```

Comment:
Maybe we should rename this to COMPANY? Would that be a more intuitive name?

Reply:
parameter name changed to COMPANY_NAME. Thanks for the heads up!